### DISCUSSION TITLE
Does the Laragon installer File have a virus?

### DISCUSSION THREAD

**Discussion by mrbarua** [leokhoa/laragon#734] [Q&A]
Does the Laragon installer File have a virus?
I found 2 malware when I checked files on the website Virustotal. 
![2024-03-23 19_59_33-Window](https://github.com/leokhoa/laragon/assets/24254733/165bee1d-d441-426d-af22-5561b2495e9e)

**Comment by leokhoa** [comment]
@mrbarua 
Laragon includes ngrok (so user can share publicly to the Internet). The included version of ngrok is marked as Trojan.WinGo.Agent but it is no harm.
Next version of Laragon will not include ngrok.

The pretty URL feature of Laragon needs to update hosts file. I will re-consider having this feature or not.

**Comment by leokhoa** [comment]
Laragon 8.0.0 has a digital signature, identified that it is created by me. It should fix the false positive issue. https://github.com/leokhoa/laragon/discussions/1058
https://github.com/leokhoa/laragon/releases/tag/8.0.0
